.github/workflows/release.yml

```diff
@@ -31,26 +31,19 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          . .github/workflows/release_info.sh ${{ github.event.repository.name }}
+          . .github/workflows/release_info.sh ${{ github.event.repository.full_name }}
```

Comment:
was testing in my fork, this provides `cfieber/fiat` instead of just `fiat`

Reply:
Ah nice change.